### PR TITLE
Add s3-insecure for S3 backend

### PR DESCRIPTION
**Pull request by ishigoya**
#655

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -167,6 +167,11 @@ If you omit the keys or the endpoint URL, then Takahē will try to use implicit
 authentication for them. The keys, if included, should be urlencoded, as AWS
 secret keys commonly contain eg + characters.
 
+With the above forms, Takahē connects to an S3 bucket using **HTTPS**. If you
```
Comment (andrewgodwin) [inline docs/installation.rst:170]: Could you call them "examples" rather than forms?

Comment (ishigoya) [reply docs/installation.rst]: done!

**Review by andrewgodwin — APPROVED** [review]

**Comment by andrewgodwin** [comment]
The linter test failed - looks like you have trailing whitespace on those lines. You can fix it by either installing and running the pre-commit hooks (`pre-commit install; pre-commit run -a`) or by manually trimming it off.

**Comment by andrewgodwin** [comment]
Thanks!

**Pull request merged**